includes/DomainCheckJob.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Miraheze\RequestSSL;
+
+use GenericParameterJob;
+use Job;
+use MediaWiki\MediaWikiServices;
+use User;
```

Comment:
```suggestion
use MediaWiki\User\User;
```

Reply:
Done in [5ab5683](https://github.com/miraheze/RequestSSL/pull/40/commits/5ab56835c8d4a262c9f8889b3eb39d98c646357d)